ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -678,6 +678,9 @@ Rectangle {
     }
 
     function onRelease(event) {
+        if (event.modifiers !== Qt.NoModifier)
```

Comment:
WDYT of checking if the event has text?
```suggestion
        if (!event.text)
```
`if (event.modifiers !== Qt.NoModifier)` Will add a regression when typing text using modifiers (Shift, Option).

Reply:
Ah ok... will check it again but the point is that checking for text won't work, as that is non-empty (contains that non-printable character).

Reply:
ohh, right. I was checking on Mac where there is no text

Reply:
Fixed, thanks!